### PR TITLE
Feature/rename in test scripts

### PR DESCRIPTION
**Pull request by achocron**
Don't merge until the "start ui tests" build finishes and passes (build # 1856)
1856 failed, but rebuilt and now it gets farther. Suspicious...

### CODE REVIEW

File: Tests/UITests/packages.config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NUnit" version="2.6.4" targetFramework="net45" />
-  <package id="Xamarin.UITest" version="2.0.3" targetFramework="net45" />
+  <package id="Xamarin.UITest" version="2.2.1" targetFramework="net45" />
```
Comment (guperrot) [inline Tests/UITests/packages.config]: isn't that the package that we had to pin version?

Comment (achocron) [reply Tests/UITests/packages.config]: I don't remember that conversation. But now if we use 2.0.3 it fails and says to use the newer version.

**Comment by guperrot** [comment]
@achocron are you running CI against this branch to test?

**Comment by achocron** [comment]
@guperrot yeah i did and it failed the first time and passed the second

**Pull request merged**